ci-operator/step-registry/sandboxed-containers-operator/peerpods/param-cm/sandboxed-containers-operator-peerpods-param-cm-commands.sh

```diff
@@ -12,6 +12,30 @@ cd /tmp || exit 1
 # can reference it.
 PP_CONFIGM_PATH="${SHARED_DIR:-$(pwd)}/peerpods-param-cm.yaml"
 
+# Create a SSH keys pair. The public key is exported and later set in
+# the peerpods-param-cm.
+#
+create_ssh_key() {
+
+    # The following was copied from the ipi-config-sshkey step
+    #
+    # Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+    # to be able to SSH.
+    if ! whoami &> /dev/null; then
+        if [[ -w /etc/passwd ]]; then
+            echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+        else
+            echo "/etc/passwd is not writeable, and user matching this uid is not found."
+            exit 1
+        fi
+    fi
+
+    local key_file="/tmp/id_rsa"
+    ssh-keygen -t rsa -f "${key_file}" -N ""
```

Comment:
perhaps we should use `id_ed25519`

Reply:
Anyway in general this approach should work provided we add the support for AZURE_SSH_KEY_PUB to openshift-tests... (unless you find out we don't need it at all)